demo/api/src/products/entities/product.entity.ts

```diff
@@ -147,9 +147,9 @@ export class Product extends BaseEntity<Product, "id"> implements DocumentInterf
     @Field(() => ProductDimensions, { nullable: true })
     dimensions?: ProductDimensions = undefined;
 
-    @Embedded(() => ProductPackageDimensions, { nullable: true })
-    @Field(() => ProductPackageDimensions, { nullable: true })
-    packageDimensions?: ProductPackageDimensions = undefined;
+    @Embedded(() => ProductPackageDimensions)
+    @Field(() => ProductPackageDimensions)
+    packageDimensions: ProductPackageDimensions;
```

Comment:
I make the field non-nullable as:
- it is broken currently, as the database fields (of the embedded) are non-nullable
- it simplifies the form; I'd discuss the simpler one first